index.html

```diff
@@ -76,7 +81,13 @@ <h2>Find a flight!</h2>
       </form>
     </section>
   </main>
+<<<<<<< HEAD
+  <footer>
+    <p> @Copyright Dominator Airlines!.......#DominateFlights.....</p>
+  </footer>
```

Comment:
stolley told us to stay away from adding a bunch of stuff all over the place, so we don't need a footer to declare some copyright that we don't have. so dont put footers anywhere and remove this one